### PR TITLE
Production yaml:Infra deployment on vsphere

### PR DESCRIPTION
**Pull request by shylesh**
Signed-off-by: Shylesh Kumar Mohan <shmohan@redhat.com>

### CODE REVIEW

File: conf/deployment/vsphere/upi_1az_rhcos_vmfs_3m_3w_3i.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  worker_replicas: 6
+  master_replicas: 3
+  worker_num_cpus: '16'
+  master_num_cpus: '4'
+  master_memory: '16384'
+  compute_memory: '65536'
+  fio_storageutilization_min_mbps: 10.0
+REPORTING:
+  polarion:
+    deployment_id: 'xxx'
```
Comment (clacroix12) [inline conf/deployment/vsphere/upi_1az_rhcos_vmfs_3m_3w_3i.yaml:19]: Is there an actual polarion-id? If not let's remove this section.

These also need empty lines at the end of the file.

Comment (petr-balogh) [reply conf/deployment/vsphere/upi_1az_rhcos_vmfs_3m_3w_3i.yaml:19]: Or have it commented out with TODO to add those polarion IDs and please open issue to not forger about it

Comment (shylesh) [reply conf/deployment/vsphere/upi_1az_rhcos_vmfs_3m_3w_3i.yaml]: Done, added 2 polarion tests and updated with test ids

**Review by clacroix12 — APPROVED** [review]

**Review by wusui — APPROVED** [review]
looks good to me

**Comment by shylesh** [comment]
./bin/oc get nodes --all-namespaces --kubeconfig  ~/shmohanprod/auth/kubeconfig 
NAME              STATUS   ROLES          AGE   VERSION
compute-0         Ready    infra,worker   60m   v1.19.0+7070803
compute-1         Ready    infra,worker   60m   v1.19.0+7070803
compute-2         Ready    infra,worker   60m   v1.19.0+7070803
compute-3         Ready    worker         60m   v1.19.0+7070803
compute-4         Ready    worker         60m   v1.19.0+7070803
compute-5         Ready    worker         59m   v1.19.0+7070803
control-plane-0   Ready    master         67m   v1.19.0+7070803
control-plane-1   Ready    master         67m   v1.19.0+7070803
control-plane-2   Ready    master         67m   v1.19.0+7070803


Verified locally

**Review by petr-balogh — APPROVED** [review]

**Review by Shrivaibavi — APPROVED** [review]
LGTM

**Pull request merged**